conda_smithy/cirun_utils.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from cirun import Cirun
+
+CIRUN_INSTALLATION_ID = os.environ.get("CIRUN_INSTALLATION_ID", 18453316)
+
+
+def enable_cirun_for_project(owner, repo):
+    """Enable the cirun.io Github Application for a particular repository."""
+    print(f"Enabling cirun for {owner}/{repo} ...")
+    cirun = _get_cirun_client()
+    assert CIRUN_INSTALLATION_ID
+    return cirun.set_repo(
+        f"{owner}/{repo}", installation_id=CIRUN_INSTALLATION_ID
+    )
+
+
+def add_repo_to_cirun_resource(owner, repo, resource, cirun_policy_args):
+    """Grant access to a cirun resource to a particular repository, with a particular policy."""
+    cirun = _get_cirun_client()
+    policy_args = {"pull_request": False}
+    if cirun_policy_args:
```

Comment:
What's the expected type/schema of `cirun_policy_args`? A list of str? A mapping?